templates/repo/create.tmpl

```diff
@@ -75,6 +75,7 @@
 							<input type="hidden" name="license" value="{{.license}}">
 							<div class="default text">{{.i18n.Tr "repo.license_helper"}}</div>
 							<div class="menu">
+								<div class="item" data-value="">No License</div>
```

Comment:
Should I add an entry to the i18n file for this, or is there some other default wording that would work?